akd/Cargo.toml

```diff
@@ -28,13 +28,17 @@ runtime_metrics = []
 parallel_vrf = ["akd_core/parallel_vrf"]
 # Parallelize node insertion during publish
 parallel_insert = []
+# Enable pre-loading of the nodes when generating history proofs
+preload_history = []
+# TESTING ONLY: Artifically slow the in-memory database (for benchmarking)
+slow_internal_db = []
```

Comment:
Interesting idea! Do we want to keep this in here long-term as a toggle, or was this meant to be removed later?

Reply:
I debated actually, if we're doing optimizations it might be helpful to say minimize the number of db calls overall if they have a non-zero cost compared to caching and transaction I/O. I feel like this might be helpful in the future....